denizen_scripts/survival/repo-link/spawn_scripts/tutorial.dsc

```diff
@@ -63,28 +112,30 @@ tutorial_data:
       - "<&e>You can manage everything related to chunks in the GUI."
       - "<&a>Command<&co> <&b>/claims"
     particle_guide: true
-  8:
-    hologram:
-      - "<&d><&l>Soul Forge"
-      - "<&e>Here you can forge your items with souls."
-      - "<&c>Souls can be acquired in the <&4>Savage Lands."
-    particle_guide: true
-  9:
+  16:
     hologram:
-      - "<&d><&l>The World (1/2)"
+      - "<&6><&l>The World (1/2)"
       - "<&e>The world is split into 2 sections<&co>"
-      - "<&e>The &cSavage Lands &eare the inner 20,000 blocks."
-      - "<&e>The &2Outer Realms &eare everywhere outside of that."
+      - "<&e>The <&c>Savage Lands <&e>are the inner 20,000 blocks."
+      - "<&e>The <&2>Outer Realms <&e>are everywhere outside of that."
     particle_guide: true
-  10:
+  17:
     hologram:
-      - "<&d><&l>The World Part (2/2)"
+      - "<&6><&l>The World Part (2/2)"
       - "<&e>The <&c>Savage Lands &ehave <&c>PvP Enabled."
```

Comment:
This line has an unparsed color code `&e`.

Reply:
Fixed.